policyengine_us/parameters/gov/states/nd/tax/income/taxable_income/subtractions/sources.yaml

```diff
@@ -6,9 +6,17 @@ values:
     - nd_qdiv_subtraction  # qualified dividends
     - taxable_social_security
 
+  2023-01-01:
+    - us_govt_interest
+    - nd_ltcg_subtraction  # long-term capital gains
+    - nd_qdiv_subtraction  # qualified dividends
+    - taxable_social_security
+    - military_service_income
```

Comment:
check 2021 - military service income was also deducted for 2022

Reply:
In the tax form instructions it says "Beginning with tax year 2023, a new individual income tax deduction is provided for all military pay. Previously, deductions were only allowed for military pay related to service members mobilized for active duty under Title 10 orders and for military retirement pay. " This is a new subtraction began on 2023.

Reply:
Hmm does that square with this from the 2022 form? Can you please add a legal code reference?

>Line 14 - Military retirement
benefit exclusion
If you are a retired military service
member, or a surviving spouse of
a deceased retired military service
member, enter on this line the amount
of taxable military retirement benefits
that you reported on Form 1040 or
Form 1040-SR, line 5b. Only include
benefits received as a retired member
of the U.S. armed forces or its reserve
components, Army National Guard, or
Air Force National Guard. “U.S. armed
forces” means the Army, Navy, Air
Force, Marine Corps, and Coast Guard.
Do not enter on this line retirement
benefits received for federal civil
service employment as a military
technician (dual status). Include a
copy of the Form 1099-R from the
Defense Finance and Accounting
Service.

Reply:
Looking through the forms, I see what happened:
In [2023](https://www.tax.nd.gov/sites/www/files/documents/forms/individual/2023-iit/2023-individual-income-tax-booklet.pdf#page=15), north dakota added a _military pay deduction_ which means that all active duty military pay is now excluded too, new line 10
![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/cbb1350c-bb30-44af-b8bd-adb4d66457a7)


However, retirement military benefits have been excluded at least since 2021

Reply:
yes, we missed `military_retirement_pay` in the original development

starting in 2023 we also add `military_service_income`

Looking through the legal code for anything else